apps/api/src/app/events/events.controller.ts

```diff
@@ -16,16 +17,7 @@ export class EventsController {
   @UseGuards(JwtAuthGuard)
   @Post('/trigger')
   trackEvent(@UserSession() user: IJwtPayload, @Body() body: TriggerEventDto) {
-    const subscribers = Array.isArray(body.to) ? body.to : [body.to];
-    const mappedSubscribers: ISubscribersDefine[] = subscribers.map((subscriber) => {
-      if (typeof subscriber === 'string') {
-        return {
-          subscriberId: subscriber,
-        };
-      } else {
-        return subscriber;
-      }
-    });
+    const mappedSubscribers = this.mapSubscribers(body);
```

Comment:
Maybe worth moving this validation to the trigger event use case rather checking it in the controller? So in case we will have other entry points for it we won't loose the validation.

Reply:
Agree :)